scripts/ci/batch_pw_load.sh

```
@@ -0,0 +1,149 @@
+setup_docker_network() {
+    docker network create "$NETWORK_NAME" 2>/dev/null || :
+}
+
+start_prod_dind() {
+    setup_docker_network
+    make build-prod
+    docker compose ${COMPOSE_ARGS} up -d --wait prod
+}
+run_make_with_prod_dind() {
+    target=$1
+    description=$2
```

Shellcheck (0.11.0)

[warning] description appears unused. Verify use (or export if used externally).

(SC2034)